Documentation/gettingstarted/cli-connectivity-test.rst

```diff
@@ -1,16 +1,9 @@
-Deploy the connectivity test
-----------------------------
-
 Run the following command to validate that your cluster has proper network
 connectivity:
 
 .. code-block:: shell-session
 
-   cilium connectivity test
-
-The output should be similar to the following one:
-
-::
+   $ cilium connectivity test
```

Comment:
Nit: blank line between command and output can likely be removed.

Reply:
Fixed in latest push.

Reply:
Actually, blank line cannot be removed, it generates a parsing error:
```
gettingstarted/k8s-install-validate.rst <included from /src/Documentation/gettingstarted/kube-router.rst>:12: WARNING: Error in "code-block" directive:
maximum 1 argument(s) allowed, 5 supplied.

.. code-block:: shell-session
   $ cilium connectivity test

   ℹ️  Monitor aggregation detected, will skip some flow validation steps
   ✨ [k8s-cluster] Creating namespace for connectivity check...
   (...)
   ---------------------------------------------------------------------------------------------------------------------
   📋 Test Report
   ---------------------------------------------------------------------------------------------------------------------
   ✅ 69/69 tests successful (0 warnings)
make[1]: *** [Makefile:50: html] Error 1
make[1]: Leaving directory '/workspace/cilium/cilium/Documentation'
make: *** [Makefile:545: test-docs] Error 2
```
Adding it back (and the other one as well).

Reply:
My bad, of course you need the blank line. I just compared before/after and saw that there was a new line. The reason you need it is that the `.. code-block::` directive takes an argument here (`shell-session`), and that when it has an argument or options, a blank line is necessary to separate the directive from the block content. Apologies.

Reply:
Wait, but, no! What I was suggesting was not
```

.. code-block:: shell-session
   $ cilium connectivity test

   ℹ️  Monitor aggregation detected, will skip some flow validation steps
```

But instead

```

.. code-block:: shell-session

   $ cilium connectivity test
   ℹ️  Monitor aggregation detected, will skip some flow validation steps
```

The latter should work and be more correct, I think?

Reply:
Ah yes, that makes more sense! Thanks.

Reply:
Fixed in latest push.